index.html

```diff
@@ -4,15 +4,245 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
-    <title>Portifolio</title>
+    <title>Brhanu's Portifolio website</title>
+    <link rel="stylesheet" href="css/style.css">
+    <script src="https://kit.fontawesome.com/e06c5cb839.js" crossorigin="anonymous"></script>
 </head>
 <body>
-    <header>
-        logo
-    </header>
-    <section class="main-container">My head line
+    <!--Header layout Starts Here-->
+    <header class="header-container">
+        <nav class="nav-top">
+            <a href="#" class="header-logo">Welcome</a>
+            <button class="hamburger" id="menu-bar" type="button">
+               <i class="fa fa-solid fa-bars"></i>
+            </button>
+            <ul class="header-nav-desktop">
+                    <li><a href="">Portifolio</a></li>
+                    <li><a href="">About</a></li>
+                    <li><a href="">Contact</a></li>
+            </ul>
+        </nav>
+        
+        <div class="mobile-menu" id="menu-mobile">
+            <div class="menu-header">
+                <span class="close">&times;</span>
+            </div>
+            <div class="menu-body">
+                <ul>
+                    <li><a href="#work-id">Portfolio</a></li>
+                    <li><a href="#about-id">About</a></li>
+                    <li><a href="#contact-id">Contact</a></li>
+                </ul>
+            </div>
+        </div>
 
+    <!--Headline starts Here-->
+    <div class="headline">
+        <div class="header-title">
+            <h1>Hey there. I’m Brhanu. </h1>
+            <p><span class="h-span">I’m a software developer</span></p>
+        </div>
+        <div class="heder-info">
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+        </div>
```

Comment:
- [ ] I will implore you to center the content of the area I highlighted so that it would fit in the mobile version.

![image](https://user-images.githubusercontent.com/86200356/212262043-0ac19219-4d6b-47cc-96b3-70dc901bbc50.png)